incuna-sass/mixins/_font-face.sass

```diff
@@ -0,0 +1,59 @@
+// New font face mixin with more flexibility and more comprehensive browser support
+// Requires that you set the font file path for the project locally
+// Like this:
+//      @function font-url($path)
+//          @return url('../path/to/fonts/' + $path)
+@mixin font-face($style-name, $file, $family, $category: '', $woff2: false, $IE-8: false)
+    $filepath: $family + '/' + $file
+    @font-face
+        font-family: '#{$style-name}'
+
+        // Default src without IE-8 and woff2 support
+        src: font-url($filepath + ".woff") format('woff'), font-url($filepath + ".otf") format('opentype')
+
+        // Support for woff2 files is optional 
+        // (this format still has quite limited support)
+        @if $woff2
+            src: font-url($filepath + '.woff2') format('woff2'), font-url($filepath + ".woff") format('woff'), font-url($filepath + ".otf") format('opentype')
```

Comment:
gah I thought I'd done that, sorry. And yeah that outputs `src` 4 times which isn't good. Is there a way that I can reword the mixin so that it only ever picks one of those 4 options?